CHANGELOG.md

```diff
@@ -22,5 +23,5 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 - Video details Dashboard
 
-[Unreleased]: https://github.com/openfun/postsie/v0.1.0...main
+[unreleased]: https://github.com/openfun/postsie/v0.1.0...main
```

Comment:
Nope: it should match the upper anchor at line 9.

Reply:
It's a fix from my formatter when I save the file. Did not notice the change when I pushed

Reply:
I think it should be fixed or it will break the reference link.